concourse/pipelines/create-microbosh.yml

```diff
@@ -120,11 +122,14 @@ jobs:
           - -e
           - -c
           - |
-            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > bosh-secrets.yml
-            ls -l bosh-secrets.yml
+            ./paas-cf/manifests/bosh-manifest/scripts/generate-bosh-secrets.sh > generate/bosh-secrets.yml
+            ls -l generate/bosh-secrets.yml
+        outputs:
+        - name: generate
```

Comment:
I think it's easier to follow if the `outputs` are listed before the `run` because the `run` step write to directories that are created by the `outputs` definitions.  What do you think?